testing/deployer/sprawl/details.go

```diff
@@ -166,8 +175,9 @@ type appDetail struct {
 	Type                  string // server|mesh-gateway|app
 	Container             string
 	Addresses             []string
-	ExposedPort           int `json:",omitempty"`
-	ExposedEnvoyAdminPort int `json:",omitempty"`
+	ExposedPort           int            `json:",omitempty"`
+	ExposedPorts          map[string]int `json:",omitempty"`
```

Comment:
I think I'd prefer if we got rid of `ExposedPort` somehow, and then for v1 clusters we required len 1

But failing that, we should document these two and how they interact if they're both set